lnbits/settings.py

```diff
@@ -52,6 +52,7 @@ class ExtensionsSettings(LNbitsSettings):
     lnbits_extensions_default_install: List[str] = Field(default=[])
     # required due to GitHUb rate-limit
     lnbits_ext_github_token: str = Field(default="")
+    lnbits_poetry_path: str = Field(default="poetry")
```

Comment:
is this path ever different than poetry? or we can you configure it?

Reply:
It would be different if you don't have poetry in your path (can be changed through .env)